docs/notebooks/PCMSolvent.ipynb

```diff
@@ -1,36 +1,122 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.            self.radii = radii  # list of cavity radii. Normally, but not always, nucleus radii.
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MRChemSoft/vampyr/pull/107/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>